sdk/maps/azure-maps-timezone/README.md

```diff
@@ -0,0 +1,366 @@
+# Azure Maps Timezone Package client library for Python
+
+This package contains a Python SDK for Azure Maps Services for Timezone.
+Read more about Azure Maps Services [here](https://docs.microsoft.com/azure/azure-maps/)
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-timezone) | [API reference documentation](https://docs.microsoft.com/rest/api/maps/timezone) | [Product documentation](https://docs.microsoft.com/azure/azure-maps/)
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to <https://github.com/Azure/azure-sdk-for-python/issues/20691>_
+
+## Getting started
+
+### Prerequisites
+
+- Python 3.7 or later is required to use this package.
+- An [Azure subscription][azure_subscription] and an [Azure Maps account](https://docs.microsoft.com/azure/azure-maps/how-to-manage-account-keys).
+- A deployed Maps Services resource. You can create the resource via [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+
+If you use Azure CLI, replace `<resource-group-name>` and `<account-name>` of your choice, and select a proper [pricing tier](https://docs.microsoft.com/azure/azure-maps/choose-pricing-tier) based on your needs via the `<sku-name>` parameter. Please refer to [this page](https://docs.microsoft.com/cli/azure/maps/account?view=azure-cli-latest#az_maps_account_create) for more details.
+
+```bash
+az maps account create --resource-group <resource-group-name> --account-name <account-name> --sku <sku-name>
+```
+
+### Install the package
+
+Install the Azure Maps Service Timezone SDK.
+
+```bash
+pip install azure-maps-timezone
+```
+
+### Create and Authenticate the TimezoneClient
+
+To create a client object to access the Azure Maps Timezone API, you will need a **credential** object. Azure Maps Timezone client also support two ways to authenticate.
+
+#### 1. Authenticate with a Subscription Key Credential
+
+You can authenticate with your Azure Maps Subscription Key.
+Once the Azure Maps Subscription Key is created, set the value of the key as environment variable: `AZURE_SUBSCRIPTION_KEY`.
+Then pass an `AZURE_SUBSCRIPTION_KEY` as the `credential` parameter into an instance of [AzureKeyCredential][azure-key-credential].
+
+```python
+import os
+
+from azure.core.credentials import AzureKeyCredential
+from azure.maps.timezone import TimezoneClient
+
+credential = AzureKeyCredential(os.environ.get("AZURE_SUBSCRIPTION_KEY"))
+
+timezone_client = TimezoneClient(
+    credential=credential,
+)
+```
+
+#### 2. Authenticate with an Azure Active Directory credential
```

Comment:
Please replace all "Azure Active Directory" to "Microsoft Entra ID". Thanks a lot :)

Reply:
Done